policies-procedures-and-docs/mentors/week-01-mentor-onboarding.md

```diff
@@ -0,0 +1,238 @@
+
```

Comment:
I would move it to the latest cohort repo

Reply:
Moving into [OLS-8](https://github.com/open-life-science/ols-8)...

Reply:
Please, see PR #[74](https://github.com/open-life-science/ols-8/pull/74).